Slide for Reddit/VideoMediaViewController.swift

```
@@ -296,20 +310,16 @@ class VideoMediaViewController: EmbeddableMediaViewController, UIGestureRecogniz
     
     @objc func unmute() {
         if isYoutubeView {
-            _ = youtubeView.webView?.stringByEvaluatingJavaScriptFromString(script: "player.unMute();")
-            youtubeMute = false
+            // TODO: - An error is thrown when this is evaluated. Resolve why to allow error handling
+            youtubeView.webView?.evaluateJavaScript("player.unMute()") { [weak self] (_, _) in
```

As the TODO says, an error gets thrown here when `player.unMute()` is called, but the function still unmutes the player. This was always the case (stringByEvaluatingJavaScriptFromString just ignores the error too). Until we know why and can resolve it, we can't really error handle, so I don't unfortunately.